spyder/widgets/github/gh_login.py

```diff
@@ -47,51 +50,102 @@ def __init__(self, parent, username):
         lbl_html = QLabel(html.format(mark=mark, title=title))
 
         # Tabs
-        tabs = QTabWidget()
+        self.tabs = QTabWidget()
 
         # Basic form layout
         basic_form_layout = QFormLayout()
         basic_form_layout.setContentsMargins(-1, 0, -1, -1)
 
+        basic_lbl_msg = QLabel(_("For regular users, i.e. users <b>without</b>"
```

Comment:
One minor comment—I'd suggest changing "For regular users, i.e. users without two-factor..." to just read "For users without two-factor...", since ideally all users *should* be using two-factor as a security best practice, and therefore nominally a "regular" user should be one using two-factor (or at least we shouldn't normalize *not* using such). It also reads shorter and cleaner.

Reply:
I thought about that but people probably doesn't know what 2FA means. So I wanted to emphasize this is for regular users.

Reply:
What's done is done, but for the record, this is a *very* bad practice from a security education standpoint. 

Considering that most major online services (email, social network, education, cloud, etc) offer it, and Spyder's audience is likely much more technologically literate than the average user, I would dearly hope so otherwise our security education efforts thus far have been a catastrophic failure. Furthermore, it is just a *really* bad practice to be explicitly declaring *not* using 2FA to be the norm for a critical service like Github in 2018; after all its been out for *five years*.